nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -186,6 +188,15 @@ def repos_from_config_file(config_file)
               sources << { url: url, key: key }
             end
           end
+
+          # signed package sources
+          # https://learn.microsoft.com/en-us/nuget/reference/nuget-config-file#trustedsigners-section
+          doc.xpath("/configuration/trustedSigners/repository").each do |node|
```

Comment:
Sorry this bit is new to me, and the docs don't clearly speak to it. Would the repository not also have to be listed as a packageSources?